### ISSUE TITLE
Spider-Man Shattered Dimensions (231990)

### ISSUE THREAD

**Issue by russbus64** [ValveSoftware/Proton#7914]
# Compatibility Report
- Name of the game with compatibility issues: Spider-Man Shattered Dimensions
- Steam AppID of the game: 231990

## System Information
- Steam Deck
- GPU: AMD AMD Custom GPU 0405 (vangogh, LLVM 15.0.7, DRM 3.54, 6.1.52-valve16-1-neptune-61)
- Video driver version: .6 (Compatibility Profile) Mesa 23.1.3 (git-87ebaf765d)
- Kernel version: 6.1.52-valve16-1-neptune-61
- Link to full system information report as [Gist](https://gist.github.com/russbus64/3dc038f67f2420d2910b4f2fa21e2d03):
- Proton version:Default from Steam, but also tested Proton GE 9-4 and 9-5.

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

## Symptoms 
Game crashes when transparency is displayed. I wasn't able to find the root cause until changing to Proton GE 9-5, where it no longer crashed. However, it instead makes objects solid white instead. After some time it seemed like the issue "fixed" itself and now transparencies look correct again?

## Reproduction
Fastest way is to enter the tutorial and when possible, look up. If lucky you can see a frame of correct-looking transparency before the game closes.

**Comment by russbus64** [comment]
Example of incorrect transparency first seen in Proton GE 9-5. Game will not crash at this point. Managed to progress through tutorial, first mission, and part of the second mission.
![231990_20240720154050_1](https://github.com/user-attachments/assets/a43e4ef7-fab2-41f4-97c4-817794fefecf)

**Comment by russbus64** [comment]
Example (from Windows) of what transparency should look like. One might see this for a brief moment before the game crashes in Proton too.
![20240707082750_1](https://github.com/user-attachments/assets/cf695f83-447b-437b-b027-2c7741be4754)

**Comment by russbus64** [comment]
Investigating further, it seems like the crashing can happen on Proton GE 9-5 and higher as well. The real issue might be shader caching? I've deleted the existing cache and turned off caching in Steam settings on desktop mode in order to resume the first Noir level.

**Comment by russbus64** [comment]
I might have found a temporary fix? I added the game and the prefix folder as an already installed game to Lutris, and used the same Proton version for its runner. Launching the game through Lutris fixes the transparency and also no longer crashes it. Going back and launching the original game through Steam has the same behavior.

**Comment by russbus64** [comment]
Here is the gist after using Lutris. 

[New Gist](https://gist.github.com/russbus64/9e6a0574014690fd6f9e9a8ee924f390)

With help from Leopard on the Lutris Discord, it looks like it might be something in [d3d_extras](https://github.com/lutris/d3d_extras/tree/v2/x32) that is helping fix the bug.

**Comment by russbus64** [comment]
To replicate my testing further, one can do the following after adding to Lutris: Configure the game, select advanced options, go to runner options, and turn off D3D Extras. This will cause the game to start crashing again. Repeat these steps to enable it again and crashes go away.

**Comment by namquang93** [comment]
I met the crash too. The game crashed as soon as I got the black suit. Here is the PROTON_LOG:

[steam-15776173790372823040.log](https://github.com/user-attachments/files/20559842/steam-15776173790372823040.log)

**Comment by russbus64** [comment]
> I met the crash too. The game crashed as soon as I got the black suit. Here is the PROTON_LOG:
> 
> [steam-15776173790372823040.log](https://github.com/user-attachments/files/20559842/steam-15776173790372823040.log)

Did you try my steps above to try and work around the crash?

**Comment by namquang93** [comment]
Yeah after that I did install the game via Lutris and got it to work (but only in desktop mode).
